vicon2_driver/include/vicon2_driver/vicon2_driver.hpp

```diff
@@ -25,27 +25,30 @@
 #include <chrono>
 #include <vector>
 
-#include "rclcpp/time.hpp"
-
-#include "mocap_msgs/msg/marker.hpp"
-#include "mocap_msgs/msg/markers.hpp"
-#include "std_msgs/msg/empty.hpp"
+#include "ViconDataStreamSDK_CPP/DataStreamClient.h"
 
 #include "rclcpp/rclcpp.hpp"
+#include "rclcpp/time.hpp"
+#include "tf2/buffer_core.h"
+#include "tf2_ros/transform_broadcaster.h"
+#include "mocap_control/ControlledLifecycleNode.hpp"
 #include "rclcpp/node_interfaces/node_logging.hpp"
 #include "rclcpp_lifecycle/lifecycle_node.hpp"
+
+#include "std_msgs/msg/empty.hpp"
 #include "lifecycle_msgs/msg/state.hpp"
 #include "lifecycle_msgs/msg/transition.hpp"
 #include "lifecycle_msgs/srv/change_state.hpp"
 #include "lifecycle_msgs/srv/get_state.hpp"
+#include "mocap_msgs/msg/marker.hpp"
+#include "mocap_msgs/msg/markers.hpp"
 
-#include "tf2/buffer_core.h"
-#include "tf2_ros/transform_broadcaster.h"
-
-#include "DataStreamClient.h"
-
-#include "mocap_control/ControlledLifecycleNode.hpp"
-
+using std::min;
+using std::max;
+using std::string;
+using std::map;
+using std::stringstream;
+using namespace ViconDataStreamSDK::CPP;
```

Comment:
Please, avoid the `using` or `using namespace` as much as possible. I think it is always positive to know the namespace of each action when you use it.

Reply:
That's true. I use namespaces to shorten the code, but probably it's better to know where does each function comes from.